stingray/io.py

```diff
@@ -653,3 +654,31 @@ def get_btis(gtis, start_time=None, stop_time=None):
 def gti_len(gti):
     """Return the total good time from a list of GTIs."""
     return np.sum([g[1] - g[0] for g in gti])
+
+
+def save_object(object, filename):
+    """
+    Pickle a class instance.
+
+    Parameters
+    ----------
+    object: a class instance
+    filename: str (name of the file to be created)
+
+    """
+
+    filename = "stingray/data/" + filename
```

Comment:
There is another way. We can determine the user's current directory and create a data folder inside if it does not already exist, using the following commands

`if not os.path.exists(directory):
    os.makedirs(directory)`